Description: Remove unused compression-related rdiff options.
Author: Andrey Rahmatullin <wrar@wrar.name>
Bug-Debian: http://bugs.debian.org/608470
Last-Update: 2011-02-08

diff --git a/rdiff.c b/rdiff.c
index 264005e..3e4223d 100644
--- a/rdiff.c
+++ b/rdiff.c
@@ -97,8 +97,6 @@ const struct poptOption opts[] = {
     { "sum-size",    'S', POPT_ARG_INT,  &strong_len },
     { "statistics",  's', POPT_ARG_NONE, &show_stats },
     { "stats",        0,  POPT_ARG_NONE, &show_stats },
-    { "gzip",         0,  POPT_ARG_NONE, 0,             OPT_GZIP },
-    { "bzip2",        0,  POPT_ARG_NONE, 0,             OPT_BZIP2 },
     { "paranoia",     0,  POPT_ARG_NONE, &rs_roll_paranoia },
     { 0 }
 };
@@ -150,8 +148,6 @@ static void help(void) {
            "IO options:\n"
            "  -I, --input-size=BYTES    Input buffer size\n"
            "  -O, --output-size=BYTES   Output buffer size\n"
-           "  -z, --gzip[=LEVEL]        gzip-compress deltas\n"
-           "  -i, --bzip2[=LEVEL]       bzip2-compress deltas\n"
            );
 }
 
